build.gradle

```diff
@@ -445,57 +457,56 @@ clean.doFirst {
 	delete tempDir, cacheFilesMinecraft
 }
 
-tasks.build.dependsOn "compressTiny"
-tasks.build.dependsOn "tinyJar"
+tasks.build.dependsOn "compressTiny","tinyJar","v2UnmergedYarnJar"
 
-task mapYarnJar(dependsOn: ["compressTiny", "mapIntermediaryJar"]) {
+task mapYarnJar(dependsOn: [compressTiny, mapIntermediaryJar]) {
 	group = mapJarGroup
 	inputs.files downloadMcLibs.outputs.files.files
 	outputs.file(yarnJar)
 
 	//Force the task to always run
-	outputs.upToDateWhen {false}
+	outputs.upToDateWhen { false }
 
 	doLast {
 		logger.lifecycle(":mapping minecraft to yarn")
 
-		def tinyInput = new File("build/libs/yarn-tiny-${yarnVersion}.gz")
+		File tinyInput = compressTiny.output
 		mapJar(yarnJar, intermediaryJar, tinyInput, libraries, "intermediary", "named")
 	}
 }
 
-task exportMappingsOfficial(dependsOn: "downloadIntermediary") {
+task exportMappingsOfficial(dependsOn: downloadIntermediary) {
 	def composeInput = downloadIntermediary.dest
 
 	doLast {
 		logger.lifecycle(":exporting mappings")
 		String[] args = [
-			"tiny",
-			composeInput.getAbsolutePath(),	
-			"enigma",
-			file("mappings/").getAbsolutePath(),
-			"enigma",
-			file("mappings_official/").getAbsolutePath(),
-			"right"
+				"tiny",
+				composeInput.getAbsolutePath(),
+				"enigma",
+				file("mappings/").getAbsolutePath(),
+				"enigma",
+				file("mappings_official/").getAbsolutePath(),
+				"right"
 		]
 
 		new ComposeMappingsCommand().run(args)
 	}
 }
 
-task importMappingsOfficial(dependsOn: "invertIntermediary") {
-	def composeInput = invertIntermediary.fileOutput
+task importMappingsOfficial(dependsOn: invertIntermediary) {
+	def composeInput = invertIntermediary.output
 
 	doLast {
 		logger.lifecycle(":importing mappings")
 		String[] args = [
-			"tiny",
-			composeInput.getAbsolutePath(),	
-			"enigma",
-			file("mappings_official/").getAbsolutePath(),
-			"enigma",
-			file("mappings/").getAbsolutePath(),
-			"right"
+				"tiny",
+				composeInput.getAbsolutePath(),
+				"enigma",
+				file("mappings_official/").getAbsolutePath(),
+				"enigma",
+				file("mappings/").getAbsolutePath(),
+				"right"
```

Comment:
There's a lot of formatting changes in here. IMO I'd try to keep it the same as it was for a cleaner PR.

Reply:
it's fineee

Reply:
Slighly annoying, but yeah dont worry.